src/components/panel/panel.js

```diff
@@ -721,6 +768,9 @@ function MdPanelService($rootElement, $rootScope, $injector, $window) {
   /** @private {!Object<string, !MdPanelRef>} */
   this._trackedPanels = {};
 
+  /** @private {!Object<string, {panels: !Array<!MdPanelRef>, maxOpen: number}>} */
+  this._groups = {};
```

Comment:
I know that I mentioned yesterday that we should use `hasOwnProperty` when checking whether a group exists, but I think I have a better solution. If we switch this line to `this._groups = Object.create(null)`, it would create an object without a prototype, which means that we can safely use the bracket notation and can revert the `hasOwnProperty` change from yesterday.